src/python/pants/backend/jvm/tasks/coverage/jacoco.py

```diff
@@ -60,6 +61,16 @@ def create(self, settings, targets, execute_java_for_targets):
                                                    scope='jacoco')
       return Jacoco(settings, agent_path, cli_path, targets, execute_java_for_targets)
 
+  # TODO(jtrobec): deprecate these options and move them to subsystem scope
+  @staticmethod
+  def register_junit_options(register, register_jvm_tool):
+    register('--coverage-jacoco-target-filters', advanced=True, type=list,
```

Comment:
Since this is unlikely to be defined in `pants.ini`, it should probably not be marked advanced.

Also, explicitly passing a `default=[]` here would be a good idea probably.